Sources/ApodiniDelta/Schema/Schema.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  File.swift
+//  
+//
+//  Created by Eldi Cano on 20.03.21.
+//
+
+import Foundation
+
+class SchemaName: PrimitiveValueWrapper<String> {}
+class IsEnum: PrimitiveValueWrapper<Bool> {}
```

Comment:
So, the logic of registering a change in the current context node, requires that all comparable properties of an object are of different types, since I am not using some additional context keys there, but use the object identifier of the type directly. If an object would have two `String` properties, when registering the changes of the properties of that object, the change would be overwritten. By using the primitive properties with the wrapper e.g. `class SchemaName: PrimitiveValueWrapper<String> {}` guarantees unique types also for a primitive properties